src/NuGet.config

```diff
@@ -3,5 +3,6 @@
   <packageSources>
     <clear />
     <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
+    <add key="dnceng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" />
```

Comment:
Revert. We can only reference one feed in our config per security standards.

I think I figured out why this always works in CI but not VS (the missing RemoteExecutor package error). Since the CI is all command-line driven, the build tests task has its own [Nuget.config](https://github.com/dotnet/SqlClient/blob/main/src/Microsoft.Data.SqlClient/tests/NuGet.config) with a reference to this feed. So if you build via command line it always works. If you build locally via command line first, then VS, VS works because the package is cached from the command line build.